src/mmw/js/src/core/modals/views.js

```diff
@@ -5,7 +5,7 @@ var _ = require('underscore'),
     coreUtils = require('../utils.js'),
     HighstockChart = require('../../../shim/highstock'),
     Marionette = require('../../../shim/backbone.marionette'),
-    ZeroClipboard = require('zeroclipboard'),
+    clipboard = require('Clipboard'),
```

Comment:
Since this is a class, later used as `new clipboard()`, consider importing as `Clipboard` to follow JavaScript class naming conventions.